lib/OpenQA/WebAPI/Controller/API/V1/Comment.pm

```diff
@@ -72,17 +72,21 @@ sub _comments ($self) {
 =item list()
 
 Returns a list of comments for a job or a job group given its id. For each comment the
-list includes its bug references, date of creation, comment id, rendered markdown text,
+list includes its bug references, date of creation, comment id,
 text, date of update and the user name that created the comment.
 
+The optional parameter "render_markdown" if set to "1" will result in the
+rendered markdown text to be included for each comment.
+
 =back
 
 =cut
 
 sub list ($self) {
     my $comments = $self->_comments();
     return unless $comments;
-    $self->render(json => [map { $_->extended_hash } $comments->all]);
+    my $render_markdown = $self->param('render_markdown') // 0;
+    $self->render(json => [map { $_->extended_hash($render_markdown) } $comments->all]);
```

Comment:
Should we do the same in `sub text` for consistency?

Reply:
As this means only selecting a single comment I left it out, as the performance impact is not that problematic there.